src/main/java/org/folio/service/mapping/profiles/TransformationConfig.java

```diff
@@ -0,0 +1,71 @@
+package org.folio.service.mapping.profiles;
+
+import org.folio.service.mapping.profiles.loader.ElectronicAccessRelationshipsLoader;
+import org.folio.service.mapping.profiles.loader.SettingsLoader;
+
+public enum TransformationConfig {
+
+  PERMANENT_LOCATION_ID("permanentLocationId",
+                        "$.{recordType}[*].permanentLocationId",
+                        "Permanent location"),
+  MATERIAL_TYPE_ID("materialTypeId",
+                   "$.{recordType}[*].materialTypeId",
```

Comment:
Material type is just a string field , not an array. Need not change it here , but just FYI for implementation

Reply:
Array sign is set only for recordType but not for materialTypeId, for item it will look $.item[*].materialTypeId